app/budget/budget.css

```diff
@@ -130,11 +130,16 @@
 }
 
 @media print {
-  @page {
-    margin: 1cm;
-  }
+	@page {
+	margin: 1cm;
+	}
 
-  .do-not-print {
+	.do-not-print {
 		display: none;
-  }
+	}
+	
+	.do-print{
+		display: block !important;
```

Comment:
The 'important' flag should only be used in extreme situations, as they make our CSS even harder to predict. The rare times I do use them its usually related to overriding something from bootstrap that used the important flag first.

Does this not work without the important flag?

Reply:
No need of !important in any of the cases pointed out.